qiskit/opflow/primitive_ops/pauli_sum_op.py

```diff
@@ -38,11 +39,13 @@ def __init__(
             self,
             primitive: SparsePauliOp,
             coeff: Union[int, float, complex, ParameterExpression] = 1.0,
+            grouping_type: Optional[str] = None,
     ) -> None:
         """
         Args:
             primitive: The SparsePauliOp which defines the behavior of the underlying function.
             coeff: A coefficient multiplying the primitive.
+            grouping_type: The type of grouping. If None, the operator is not grouped.
```

Comment:
I prefer introducing a word such `none` to represent no-grouping rather than `None` of optional. Enum is another option.